library/tools/smt2/syntax.lean

```diff
@@ -0,0 +1,85 @@
+namespace smt2
+
+-- not sure about this one
+@[reducible] def symbol : Type := string
+@[reducible] def identifier : Type := string
+
```

Comment:
I just wasn't sure if they should be strings or not, will remove comment for time being.